TASK: Add workspaceName to nodeAggregate

mhsdesign opened the pull request:
Followup to https://github.com/neos/neos-development-collection/pull/5042


**Upgrade instructions**

<!-- 
    Add upgrade instructions for breaking changes. 
    Explain who is affected, what has to be adjusted  
-->

**Review instructions**

<!-- 
    If your change is not explained fully by the first section you can 
    add more details here to help the reviewers understand the change.
    We have to understand what you did, why you did it and how we can 
    verify it works correctly and does no harm.
-->

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

mhsdesign committed:
TASK: Add workspaceName to nodeAggregate

bwaidelich reviewed (approved):
great, +1 by reading

mhsdesign committed:
TASK: Prefer `NodeAggregate::$workspaceName` over `NodeAggregate::$contentStreamId`

Adjust tests to assert on workspace name as well:

> And I expect this node aggregate to have the parent node aggregates ["lady-eleonode-rootford"]

mhsdesign committed:
TASK: Make NodeAggregate __construct private
